synedrion/src/cggmp21/entities.rs

```diff
@@ -1,8 +1,10 @@
 use alloc::{
     collections::{BTreeMap, BTreeSet},
+    format,
     vec::Vec,
 };
 use core::{fmt::Debug, marker::PhantomData};
+use manul::session::LocalError;
```

Comment:
Not entirely sure if we should use our own error instead of reusing `manul`'s, but that'll work for now.

Reply:
Yeah I had the same thought: soon enough we'll need to have a think about which errors we want.